### PR TITLE
[ready] Makes events ghost-announce their spawned atom much more consistently

### LINKED ISSUE #41217: Ghost are now getting random event announcements AFTER stationwide CentComm announcements about them
Issue reported from  Round ID: 96265 (/tg/Station Sybil [ENGLISH] [US-EAST] [100% LAG FREE])
Reporting client version: 512
#41029: '[SUPER DUPER EXTRA READY] Hilbert's Hotel' by TheDreamweaver at commit 47b488e326

Before #40727 was merged, the ghost announces (`X has just been randomly triggered!`) were showing ahead of any centcomm announces, now they're occasionally showing below.

### PR DESCRIPTION
**Pull request by MrDoomBringer**
Fixes #41217

:cl: MrDoomBringer
fix: Events now announce follow-able atoms to ghosts much more reliably
/:cl:

Implements @ShizCalev's suggestion to just have the event ghost-announce normally, then call a separate ghost announcement for the "atom of interest" as it is created.

### CODE REVIEW

File: _maps/_basemap.dm [hunk 1]
```diff
@@ -1,4 +1,4 @@
-//#define LOWMEMORYMODE //uncomment this to load centcom and runtime station and thats it.
+#define LOWMEMORYMODE //uncomment this to load centcom and runtime station and thats it.
```
Comment (SpaceManiac) [inline _maps/_basemap.dm]: achtung

**Comment by kriskog** [comment]
After the previous one, I noticed the window now blinks on a lot of events without follow links. Are you able to prevent this?

**Comment by MrDoomBringer** [comment]
@kriskog this PR should fix that

File: code/modules/events/_event.dm [hunk 2]
```diff
@@ -130,15 +129,14 @@
...
 	return
 
-//Called after start()
-//Lets the ghosts know that the event has started, and provides a follow link to a mob if possible
+//Called after something followable has been spawned by an event
+//Provides ghosts a follow link to an atom if possible
 //Only called once.
-/datum/round_event/proc/announce_to_ghosts()
+/datum/round_event/proc/announce_to_ghosts(atom/atom_of_interest)
+	message_admins("meme")
```
Comment (AnturK) [inline code/modules/events/_event.dm]: Ahem

Comment (MrDoomBringer) [reply code/modules/events/_event.dm]: i should really start looking at my own code

**Comment by ninjanomnom** [comment]
Let one of the maintainers know when you work on this again

**Comment by MrDoomBringer** [comment]
should be good 2 go

**Comment by kriskog** [comment]
>Erick Vasilyev has just been spawned from Spontaneous Appendicitis! (Click to orbit)
Round ID 97096

**Pull request merged**